TUTORIAL.md

```diff
@@ -0,0 +1,386 @@
+        })
+    ],
+    mode: 'development',
+};
+```
+
+## Incorporate CesiumJS
+1. Updated `index.js` with CesiumJS starter code:
+```
+import { Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "../node_modules/cesium"
+import "../node_modules/cesium/Build/Cesium/Widgets/widgets.css";
+
+// Your access token can be found at: https://cesium.com/ion/tokens.
+// This is the default access token from your ion account
+Ion.defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJlYWE1OWUxNy1mMWZiLTQzYjYtYTQ0OS1kMWFjYmFkNjc5YzciLCJpZCI6NTc3MzMsImlhdCI6MTYyNzg0NTE4Mn0.XcKpgANiY19MC4bdFUXMVEBToBmqS8kuYpUlxJHYZxk';
+
+// Initialize the Cesium Viewer in the HTML element with the `cesiumContainer` ID.
+const viewer = new Viewer('cesiumContainer', {
+  terrainProvider: createWorldTerrain()
+});
+
+// Add Cesium OSM Buildings, a global 3D buildings layer.
+viewer.scene.primitives.add(createOsmBuildings());   
+
+// Fly the camera to San Francisco at the given longitude, latitude, and height.
+viewer.camera.flyTo({
+  destination : Cartesian3.fromDegrees(-122.4175, 37.655, 400),
+  orientation : {
+    heading : Math.toRadians(0.0),
+    pitch : Math.toRadians(-15.0),
+  }
+});
+```
+This code initializes the Cesium Viewer, adds Cesium OSM Buildings to the terrain, and moves the `Camera` to San Francisco. 
+2. Update `index.html` with the CesiumContainer
+```
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+</head>
+<body>
+  <div id="cesiumContainer"></div>
+</body>
+</html>
+```
+3. Run the command `npm run build`. Ensure that the project is built correctly.
+4. Run `npm start` and open http://localhost:8080/ in a web browser to see the CesiumJS viewer.
+
+![South San Francisco](./screenshots/ssf2.png)
+
+Copy and paste your favorite Sandcastle example. For example, [The Particle System Fireworks](https://sandcastle.cesium.com/?src=Particle%20System%20Fireworks.html) demo makes for a fun demo.
+
+![South San Francisco](./screenshots/fw.png)
+
+# Advanced webpack configurations and Resources
+
+Webpack can be leveraged in many more ways to increase performance, decrease your bundle size, and perform additional or complex build steps. Here we’ll discuss a few configuration options relevant to using the CesiumJS library.
+
+## Enable source maps
+
+Source maps allow webpack to trace errors back to the original content. They offer more or less detailed debugging information in exchange for compiling speed. We recommend setting `devtool` to the 'eval' option in `webpack.config.js`
+
+` devtool: 'eval' `
+
+Please note that source maps are not recommended for production code.
+
+## Additional Resources
+
+The official cesium-webpack-example  repo contains the minimal webpack configuration, the hello world code covered in this tutorial, and instructions for optional code configurations.
+
+For a tour of CesiumJS feature to include in your new app, see the Cesium Workshop Tutorial.
```

Comment:
I don't think the Cesium Workshop Tutorial still exists. If you can find it, link it here, and if not, delete this line.